### ISSUE TITLE
Is there support for RazorGenerator.Templating in VS 2017?

### ISSUE THREAD

**Issue by fschwiet** [RazorGenerator/RazorGenerator#146]
It looks like the latest version of https://www.nuget.org/packages/RazorGenerator.Templating/ was published before VS2017 support.  I have an old console application project that uses RazorGenerator.Templating (one of the console commands hosts a webserver via NancyFX).  How can I get my cshtml file compiling again?  Its just a @* Generator: Template *@ type template.

**Comment by davidebbo** [comment]
What issue are you running into specifically? I just tried my test project https://github.com/davidebbo-test/RazorGeneratorSimpleTemplate, and it's working fine in VS2017. I was able to make a change to SampleTemplate.cshtml, which generated new code, and then everything ran. Does that test app work for you?

**Comment by fschwiet** [comment]
Yes, that project is working for me.  And now my project is working...

I think honestly I may have needed to restart VS or the machine or something to get it working.  I did a diff what different in the project, made some adjustments to try to be closer to the sample project, and it started to work.  But then I reverted my changes and it still worked.  So it didn't seem like I needed to make a specific change for it to start working.


More details of what I did:

I look at my sln, csproj and packages files to see what might be different, and saw the following noted.  The MSBuildToolsVersion related import looked valuable, but I had no idea how to add that, so I tried changing the project build configs from x86->AnyCPU.  At that point things started worked.  HOWEVER, at that point it looks like something (VS?) also added the MSBuildToolsVersion import.   So I commited the working changes, closed VS, git revert'd to where it wasn't working, git clean'd everything, and tried it in VS.  I was thinking I'd do the incremental changes to see when it starts working, but it actually worked right away.

```xml
    <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
    <AppDesignerFolder>Properties</AppDesignerFolder>

    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
    not <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```
